openslides_backend/action/action.py

```diff
@@ -453,12 +453,18 @@ def execute_other_action(
             self.datastore,
             self.relation_manager,
             self.logging,
-            {**self.additional_relation_models, **additional_relation_models},
+            {
+                **self.datastore.additional_relation_models,
+                **self.additional_relation_models,
+                **additional_relation_models,
+            },
+        )
+        write_request, action_results = action.perform(
+            payload, self.user_id, internal=True
         )
-        # ignore the action result elements
-        write_request, _ = action.perform(payload, self.user_id, internal=True)
         if write_request:
             self.write_requests.append(write_request)
+        return write_request, action_results
```

Comment:
Do you need the write request anywhere? If not, I would prefer to just return the action results so the write request handling is completely contained inside this method.

Reply:
see also comments below

Reply:
I  need it, in this case to check for the admin and default group. The advantage of this one further is, that these are only the write_requests from this action and not for the whole bunch of it, we will have to look at sing them from self.write_request.